### ISSUE TITLE
Formal documentation?

### ISSUE THREAD

**Issue by bruceharrison1984** [honojs/honox#86]
### What is the feature you are proposing?

The current readme is getting pretty long. Can we get a formal documentation repo (similar to Hono) to start moving some of this stuff over?

**Comment by yusukebe** [comment]
Hi @bruceharrison1984 

We don't have to do it "right now", but we should do it soon.

I think those are the two options:

1. Add a section to Hono's website for HonoX.
2. Create a new HonoX website using "HonoX".

If we chose "1", we can add that here:

<img width="1151" alt="Screenshot 2024-02-23 at 6 09 14" src="https://github.com/honojs/honox/assets/10682/14eadcee-f898-41ed-a07d-88c4383f7aa7">

The most likely way is to do "1" first and then "2" later.

**Comment by bruceharrison1984** [comment]
@KaiSpencer I agree. I think just carving out a section in the existing docs, similar to some of the middleware, would work for quite some time
